plugins/modules/synchronize.py

```diff
@@ -606,6 +619,9 @@ def _write_password_to_pipe(proc):
     else:
         (rc, out, err) = module.run_command(cmd)
 
+    if quiet:
+        cmd.append('--quiet')
+
```

Comment:
This should be moved up to before the paths get appended. Right now, it's being added after the command has been run, so it won't take effect.